src/RegistryCoordinatorStorage.sol

```diff
@@ -64,6 +64,11 @@ abstract contract RegistryCoordinatorStorage is IRegistryCoordinator {
     /// @notice the address of the entity allowed to eject operators from the AVS
     address public ejector;
 
+    /// @notice the last timestamp an operator was ejected
+    mapping(address => uint256) public lastEjectionTimestamp;
+    /// @notice the delay before an operator can reregister after being ejected
```

Comment:
should probably do a dev tag + explain this is in seconds? or otherwise have seconds in the name (but this may be 'gross' / lead to a sentence-long name)